docs/create/heroku/herokuruby/index-herokuruby.mdx

```diff
@@ -0,0 +1,201 @@
+
+
+### Step 1. Create Redis Enterprise Cloud
+
+Create your free Redis Enterprise Cloud account by visiting [this link](https://redis.com/try-free)
+
+![recloud](try-free.png)
+
+[Follow this link to create a Redis Enterprise Cloud](/create/rediscloud) subscription and database as shown below:
+
+
+![heroku](launch_database.png)
+
+The database endpoint URL is unique for all and hence might be different in your case. Save it for future reference.
+
+
+
+### Step 2. Create a Heroku account
+
+If you are using Heroku for the first time, create your new Heroku account [through this link](https://signup.heroku.com/login)
+
+![heroku](create_heroku.png)
+
+
+### Step 3. Install Heroku CLI on your system
+
+   ```macos
+    brew install heroku
+   ```
+
+### Step 4. Login to Heroku
+
+   ```bash
+    heroku login
+    heroku: Press any key to open up the browser to login or q to exit:
+    Opening browser to https://cli-auth.heroku.com/auth/cli/browser/XXXXXXXXXXA
+    Logging in... done
+    Logged in as your_email_address
+   ```
+
+### Step 5. Connect your application to Redis Enterprise Cloud
+
+For this demonstration, we will be using a [Sample Redis Leaderboard app](https://github.com/redis-developer/basic-redis-leaderboard-demo-ruby)
+
+#### Clone the repository
+
+   ```bash
+    git clone https://github.com/redis-developer/basic-redis-leaderboard-demo-ruby
+   ```
+
+
+
+Run the commands below to get a functioning Git repository that contains a simple application as well as a package.json file.
+
+
+```
+heroku create
+Creating app... done, ⬢ thawing-shore-07338
+https://thawing-shore-07338.herokuapp.com/ | https://git.heroku.com/thawing-shore-07338.git
+```
+
+
+
+### Step 6. Setting up Environmental variable
+
+Go to the Heroku dashboard, click "Settings" and set REDIS_HOST and REDIS_PASSWORD under the Config Vars.
+Refer to Step 1 for the correct values to use.
+
+![heroku](heroku_app1_env.png)
+
+
+
+You now have a functioning Git repository that contains a simple application as well as a package.json file, which is used by Node’s dependency manager.
+
+
+
+
+### Step 7. Deploy your code
+
+
+```
+$ git push heroku
+```
+
+Wait for few seconds and you will see the messages below:
+
+```
+remote: -----> Discovering process types
+remote:        Procfile declares types     -> (none)
+remote:        Default types for buildpack -> console, rake, web
+remote: 
+remote: -----> Compressing...
+remote:        Done: 125.9M
+remote: -----> Launching...
+remote:        Released v10
+remote:        https://thawing-shore-07338.herokuapp.com/ deployed to Heroku
+remote: 
+remote: Verifying deploy... done.
+To https://git.heroku.com/thawing-shore-07338.git
+ * [new branch]      master -> master
+```
+
+### Step 8. Accessing the application
+
+
+Open https://thawing-shore-07338.herokuapp.com/    to see your application
```

Comment:
Wherever the Heroku URL is shown do we need to mention that this will be your URL from Heroku not the one shown in the tutorial here?

Reply:
Yes, you're right. Let me capture that.